### PR TITLE
Update linking-a-repository-to-a-project-board.md

### LINKED ISSUE #160: Unable to link specific repositories to projects
### Issue overview

In our documentation we do not make it clear to users that in order to link a repository to your organisation or user owned project board the repository needs to have Issues enabled. 

Help Article: https://help.github.com/en/github/managing-your-work-on-github/linking-a-repository-to-a-project-board

### Content to update

In https://help.github.com/en/github/managing-your-work-on-github/linking-a-repository-to-a-project-board:

- Add a sentence "In order to link a repository to your organization or user owned project board the repository needs to have Issues enabled"
- We should really link to an article about issues being enabled, maybe https://docs.github.com/en/github/managing-your-work-on-github/disabling-issues?

### PR DESCRIPTION
**Pull request by casals**
### Why:

Fixes #160 .

### What's being changed:

Made the changes suggested by the author, including adding the link about enabling Issues. Since this link specifically refers to a sub-topic called "Disabling Issues", I added it within an extra sentence to avoid confusion.

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks @casals! I'll let the @github/docs-content-core team know this is ready for review 🎉

File: content/github/managing-your-work-on-github/linking-a-repository-to-a-project-board.md
Comment (felicitymay) [inline content/github/managing-your-work-on-github/linking-a-repository-to-a-project-board.md:12]: > Since this link specifically refers to a sub-topic called "Disabling Issues", I added it within an extra sentence to avoid confusion.

This is a great observation. You're absolutely right to point out that a simple link that topic would have been confusing for users. ✨ 

I'm going to suggest an alternative fix because we actively avoid using this style of link (if you're interested in why, see [Make link text meaningful](https://readabilityguidelines.co.uk/content-design/links/#1-make-link-text-meaningful)). We also avoid mid-sentence links for accessibility and readability (see the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#links).

```suggestion
**Note:** In order to link a repository to your organization or user owned project board the repository needs to have issues enabled. For information on how to see if issues are disabled for a repository, see "[Disabling issues](/github/managing-your-work-on-github/disabling-issues) ."
```

**Review by felicitymay — CHANGES_REQUESTED** [review]
It's great having your help resolving issues and making the docs clearer for other users.

One change required here and then this is ready to merge as well 🎉

File: content/github/managing-your-work-on-github/linking-a-repository-to-a-project-board.md
Comment (casals) [reply content/github/managing-your-work-on-github/linking-a-repository-to-a-project-board.md:12]: Thanks for the reference!

Comment (felicitymay) [reply content/github/managing-your-work-on-github/linking-a-repository-to-a-project-board.md]: No trouble

**Comment by felicitymay** [comment]
As soon as I've got the next PR on my list merged, I'll circle back and update/merge this PR too 😄

**Review by felicitymay — APPROVED** [review]
Thanks for the fix.

**Pull request merged**
